.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "gnu-efi"]
+	path = gnu-efi
+	url = https://git.code.sf.net/p/gnu-efi/code
```

Comment:
I think we should just use the shim specific `gnu-efi` here instead of doing it as a follow-up separate patch.

Also, this patch has your authorship but mjg's Signed-off-by tag.